docs/queries.rst

```diff
@@ -720,3 +731,27 @@ Properties
 .. php:attr:: defaultField
 
     If no fields are defined, this field is used.
+
+.. php:attr:: template_select
```

Comment:
They nee to be moved into appropriate sections. I'll move it, as i wanted to do it yesterday anyway but didn't have time.